src/main/resources/edu/cornell/kfs/ksr/workflow/SecurityProvisioningMaintenanceDocument.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<data xmlns="ns:workflow" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="ns:workflow resource:WorkflowData">
+  <documentTypes xmlns="ns:workflow/DocumentType" xsi:schemaLocation="ns:workflow/DocumentType resource:DocumentType">
+    <documentType>
+      <name>SecurityProvisioningGroupDocumentType</name>
+      <parent>FSSM</parent>
+      <label>Security Provisioning</label>
+      <docHandler>${application.url}/maintenance?methodToCall=docHandler</docHandler>
```

Comment:
Can you please confirm whether this is the correct URL format? Does it need to reference "maintenance.do" instead? The same goes for the Security Group doc type from the previous KSR pull request.

Reply:
Great question! The current SecurityGroupDocumentType in  PROD has:
Document Handler URL: ${kr.krad.url}/maintenance?methodToCall=docHandler

But I can see that the ACCT doc type has the URL with maintenance.do:
Document Handler URL: ${kfs.url}/kr/maintenance.do?methodToCall=docHandler (Inherited from Parent)

Perhaps this is because SecurityGroupDocumentType was built using s different framework?

Reply:
Yes, the difference is likely because the Cynergy KSR documents are currently using the KRAD UI setup instead of KNS.